src/worker.ts

```diff
@@ -1,3 +1,14 @@
-export function init(): void {
-  console.log("Hello from Worker!"); // eslint-disable-line no-console
+import "unfetch/polyfill";
+import "native-promise-only";
```

Comment:
I like that you've thought about this, however we are already feature detecting `fetch` support within the loader [`index.ts`](https://github.com/fastly/insights.js/blob/v2.0.0/src/index.ts#L6). So no need to include the polyfills. We should also double check that babel-preset-env isn't adding a core-js polyfill for fetch.

Reply:
I thought that might be the case; so we can dispense with both the `fetch` and the `promise` polyfills? My node `v10` implementation doesn't have `fetch` though, so tests won't pass without it. Do you prefer `node-fetch`?

Reply:
I believe Jest uses jsdom by default which does have a `fetch` implementation, so tests should work.

Reply:
Sadly, this is not the case
![screenshot](https://cl.ly/9d41d13acdfa/Image%2525202019-06-05%252520at%25252010.27.32.png)